From 7821ecc0b35c422bc8ed26e0e44c841d067f88d7 Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Tue, 24 Apr 2018 18:27:25 +0200
Subject: [PATCH] test/udev-test.pl: merge "space and var with space" tests

As we can check multiple links in a single test now, these 3
tests can be merged into one.

(cherry picked from commit 2084fe0d3290c525ecb9faa07d07c3abc2488e59)

Related: #1642728
---
 test/udev-test.pl | 31 +++----------------------------
 1 file changed, 3 insertions(+), 28 deletions(-)

diff --git a/test/udev-test.pl b/test/udev-test.pl
index 6928439d14..880a73b10b 100755
--- a/test/udev-test.pl
+++ b/test/udev-test.pl
@@ -1103,34 +1103,9 @@ EOF
                 devices => [
                         {
                                 devpath         => "/devices/pci0000:00/0000:00:1d.7/usb5/5-2/5-2:1.0/tty/ttyACM0",
-                                exp_links        => ["first"],
-                                not_exp_links    => [" "],
-                        }],
-                rules           => <<EOF
-ENV{WITH_WS}="   one  two  three   "
-SYMLINK="  first  name-\$env{WITH_WS}-end another_symlink a b c "
-EOF
-        },
-        {
-                desc            => "symlink with space and var with space, part 2",
-                devices => [
-                        {
-                                devpath         => "/devices/pci0000:00/0000:00:1d.7/usb5/5-2/5-2:1.0/tty/ttyACM0",
-                                exp_links        => ["name-one_two_three-end"],
-                                not_exp_links    => [" "],
-                        }],
-                rules           => <<EOF
-ENV{WITH_WS}="   one  two  three   "
-SYMLINK="  first  name-\$env{WITH_WS}-end another_symlink a b c "
-EOF
-        },
-        {
-                desc            => "symlink with space and var with space, part 3",
-                devices => [
-                        {
-                                devpath         => "/devices/pci0000:00/0000:00:1d.7/usb5/5-2/5-2:1.0/tty/ttyACM0",
-                                exp_links        => ["another_symlink"],
-                                not_exp_links    => [" "],
+                                exp_links       => ["first", "name-one_two_three-end",
+                                                    "another_symlink", "a", "b", "c"],
+                                not_exp_links   => [" "],
                         }],
                 rules           => <<EOF
 ENV{WITH_WS}="   one  two  three   "
